### PR TITLE
CCXDEV-5797 Insights Operator 4.9 Release Notes

### PR DESCRIPTION
**Pull request by xJustin**
This applies only to `enterprise-4.9` 

Jira: https://issues.redhat.com/browse/CCXDEV-5797 

Bugzillas: 

- https://bugzilla.redhat.com/show_bug.cgi?id=1977753 Gather all MachineConfig definitions
- https://bugzilla.redhat.com/show_bug.cgi?id=2000820 Gather PodSecurityPolicies names installed in a cluster
- https://bugzilla.redhat.com/show_bug.cgi?id=1986833 Gather Openshift Logging Stack Data 

Preview: 

https://deploy-preview-36247--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-insights-operator

### CODE REVIEW

**Comment by xJustin** [comment]
@tremes and @radekvokal can you take a look when you get a chance?

**Comment by tremes** [comment]
LGTM thanks @xJustin!

**Comment by xJustin** [comment]
Thanks! @quarckster this one is ready for QE.

**Comment by sagidlow** [comment]
LGTM

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -191,6 +191,25 @@ For more information, see xref:../scalability_and_performance/using-topology-man
 [id="ocp-4-9-insights-operator"]
 === Insights Operator
 
+[id="ocp-4-8-insights-operator-restricted-network"]
```
Comment (quarckster) [inline release_notes/ocp-4-9-release-notes.adoc:194]: why the id refers to `4.8`?

**Review by quarckster — APPROVED** [review]
LGTM

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (xJustin) [reply release_notes/ocp-4-9-release-notes.adoc]: good catch! Fixed

**Pull request merged**